plugins/kopilot.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kopilot
+spec:
+  version: v0.0.2
+  homepage: https://github.com/knight42/kopilot
+  shortDescription: Your AI Kubernetes Expert
```

Comment:
This is not descriptive enough and many plugins in the future may say exactly the same thing, can you elaborate or be more specific?

Reply:
Updated. PTAL